utils/person.js

```diff
@@ -0,0 +1,19 @@
+import {NativeModules} from 'react-native';
+import {getLocations} from '../realm/realmLocationTasks';
+import { getContactsByDate } from '../realm/realmPersonTasks';
+
+import { fetchContactsByDate, addPerson } from '../realm/realmPersonTasks';
+
+const Person = {
+  fetchContactsByDate: async function(dateObj, dayRange = 0) {
```

Comment:
fetchContactsByDate and savePerson seems just to be the wrapper of realm task functions. Can we call getContactsByDate and addPerson in the component directly without pulling them to a util function?

Reply:
I was thinking that this can have a good separation of concerns. Since the utils we can further add more business logic like filters or invalidation logic before saving something to realm database versus the database operations is just for persistence.